charts/stateless/values.yaml

```diff
@@ -230,3 +230,8 @@ extraContainers:
   #   command: ['do', 'something']
 # Allows you to add any config files in /usr/share/myapp/config/
 # as a ConfigMap
+
+keda:
```

Comment:
have defaults for keda cpu scaler here. This is needed when running helm charts from cli etc.
(same as what is present in the UI right now)